web/frontend/src/components/slam/index.svelte

```diff
@@ -66,6 +73,9 @@ $: unitScale = labelUnits === 'm' ? 1 : 1000;
 
 // get all resources which are bases
 $: bases = filterSubtype($components, 'base');
+$: slamResourceName = filterSubtype($services, 'slam').find(
+  (s) => s.name === name
```

Comment:
```suggestion
  (service) => service.name === name
```

Surprised our linter didn't catch this but we usually don't allow single-letter variables names.